src/requests.js

```diff
@@ -24,3 +24,11 @@ export const getReviews = (category) => {
             return res;
         })
 };
+
+export const patchReview = (reviewId) => {
+    return gamesApi
+        .patch(`/reviews/${ reviewId }`, { inc_votes: reviewId })
```

Comment:
You are increasing the votes of the article by their reviewid. So the article with reviewId 10 has an increase of 10 likes etc.